packages/lab/stories/toggle-button-group/toggle-button-group.doc.stories.mdx

```diff
@@ -0,0 +1,117 @@
+import { ComponentsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { ToggleButton, ToggleButtonGroup } from "@salt-ds/lab";
+import HelpAndSupport from "docs/blocks/help-and-support.doc.mdx";
+
+<Meta
+  title="Documentation/Lab/Toggle Button Group"
+  component={ToggleButtonGroup}
+  subcomponents={{ ToggleButton }}
+  parameters={{
+    viewMode: "docs",
+  }}
+/>
+
+# Toggle Button Group
+
+<Canvas>
+  <Story id="lab-toggle-button-group--horizontal" />
+</Canvas>
+
+The Toggle Button Group allows users to make a mutually exclusive selection from a set of related commands—with only one option selected at a time.
+
+## Examples
+
+### Text and Icon
+
+Use icons and text in Toggle Buttons if it's important to visually reinforce the theme of the options or if providing additional context may help decision-making (for examples, selecting a document type or a text style).
+
+- Pass `aria-hidden=”true”` to the icon so its label is not announced by the screen reader.
+- To reduce visual noise and mitigate alignment issues, use icons in Toggle Button labels consistently across all options in the group.
+
+<Canvas>
+  <Story id="lab-toggle-button-group--horizontal" />
```

Comment:
on focus, the icon and text have a different color, is that intended?

Reply:
I think it's up to design to push this but the focus styling should include hover styling